common/types/string.go

```diff
@@ -497,3 +523,225 @@ func parseClausePrecision(lastStrIndex int, formatStr string) (int, int, error)
 func isASCIIDigit(r rune) bool {
 	return r <= unicode.MaxASCII && unicode.IsDigit(r)
 }
+
+func matchLanguage(locale string) language.Tag {
+	tag, _ := language.MatchStrings(langMatcher, locale)
+	return tag
+}
+
+func initTags() []language.Tag {
+	// this is used for initializing locales whose behavior does not match
+	// the default (en_US) - if this function is used for other purposes
+	// and/or localization is used for datatypes besides floating-point
+	// numbers, this list may need to be updated
+	// golang's default catalog/matcher is empty, so we need to build our own
```

Comment:
Use a function name that hints at the limits? Move this comment up to the function's godoc?

Reply:
Makes sense - I've moved the comment to the godoc and chosen a more descriptive name.